### ISSUE TITLE
[Question] Do self-hosted invidious work over vpn connections? 

### ISSUE THREAD

**Issue by 8mpty** [iv-org/invidious#4399]
My configuration goes as follows:

My current network is behind a CGNAT but I would still like to access my locally hosted services over the Internet. So I made a wireguard VPN server on a Google Cloud VM it it just runs the Wireguard server.

On my local network:
I have a Promox server that runs a VM with Portainer and Invidious hosted inside it. I am able to access the Invidious instance on my local network (over something like 192.168.1.10:3000). If I connect my Invidious VM to the wireguard server using a configuration file generated like (wg1.conf) and my phone with (wg2.conf), I would not be able to access the Invidious instance anymore. The status for it in portainer would also be unhealthy. Why is that so actually but accessing the portainer webui over the VPN works?

TLDR;
Local:
Phone(192.168.1.11) -> able to access -> Invidious(192.168.1.10:3000) 

Over Wireguard VPN:
Phone(10.10.10.1) -> not able to access -> Invidious(10.10.10.2:3000) (But is able to ping each other)
Phone(10.10.10.1) -> able to access -> Portainer(10.10.10.2:9443)

My Invidious Config is:
```
version: "3"
services:

  invidious:
    image: quay.io/invidious/invidious:latest
    # image: quay.io/invidious/invidious:latest-arm64 # ARM64/AArch64 devices
    restart: unless-stopped
    ports:
      - "3000:3000"
    environment:
      # Please read the following file for a comprehensive list of all available
      # configuration options and their associated syntax:
      # https://github.com/iv-org/invidious/blob/master/config/config.example.yml
      INVIDIOUS_CONFIG: |
        db:
          dbname: invidious
          user: kemal
          password: kemal
          host: invidious-db
          port: 5432
        check_tables: true
        # external_port:
        # domain:
        # https_only: false
        # statistics_enabled: false
        registration_enabled: false
        # local: true
        # host_binding: 0.0.0.0
        hmac_key: ${HMAC}
    healthcheck:
      test: wget -nv --tries=1 --spider http://127.0.0.1:3000/api/v1/comments/jNQXAC9IVRw || exit 1
      interval: 30s
      timeout: 5s
      retries: 2
    logging:
      options:
        max-size: "1G"
        max-file: "4"
    depends_on:
      - invidious-db

  invidious-db:
    image: docker.io/library/postgres:14
    restart: unless-stopped
    volumes:
      - postgresdata:/var/lib/postgresql/data
      - ./config/sql:/config/sql
      - ./docker/init-invidious-db.sh:/docker-entrypoint-initdb.d/init-invidious-db.sh
    environment:
      POSTGRES_DB: invidious
      POSTGRES_USER: kemal
      POSTGRES_PASSWORD: kemal
    healthcheck:
      test: ["CMD-SHELL", "pg_isready -U $$POSTGRES_USER -d $$POSTGRES_DB"]

volumes:
  postgresdata:
```

**Comment by unixfox** [comment]
Hello,

For these types of not so specific to invidious. I would suggest asking on the matrix room (https://matrix.to/#/#invidious:matrix.org) or IRC channel (https://web.libera.chat/?channel=#invidious).

I'm going to close this issue because the invidious maintainers do not have time to reply to your question not specifically related to invidious, but a more general networking question.
